fe/fe-core/src/main/java/org/apache/doris/backup/BackupHandler.java

```diff
@@ -408,6 +408,10 @@ private void backup(Repository repository, Database db, BackupStmt stmt) throws
     private void restore(Repository repository, Database db, RestoreStmt stmt) throws DdlException {
         BackupJobInfo jobInfo;
         if (stmt.isLocal()) {
+            if (stmt.getJobInfo() == null) {
```

Comment:
check this in Restore analyze, maybe u need to mark from sql or thrift in RestoreStmt